components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -86,6 +92,16 @@ export default class GroupDetails extends React.PureComponent {
         });
     }
 
+    componentDidUpdate(prevProps) {
+        /* eslint-disable react/no-did-update-set-state */
+        if (this.props.groupTeams !== prevProps.groupTeams) {
+            this.setState({groupTeams: this.props.groupTeams});
+        }
+        if (this.props.groupChannels !== prevProps.groupChannels) {
+            this.setState({groupChannels: this.props.groupChannels});
+        }
+    }
```

Comment:
According to [the React docs](https://reactjs.org/docs/react-component.html#componentdidupdate) this is fine, not sure why eslint is configured to not allow it.